src/main/resources/com/google/api/codegen/go/mock.snip

```diff
@@ -127,3 +174,58 @@
         return err
     }
 @end
+
+@private testError(test)
+    func {@test.nameWithException}(t *testing.T) {
+        expErr := fmt.Errorf("test err %d", rand.Int())
+        {@test.mockServiceVarName}.err = expErr
+
+        c, err := {@test.serviceConstructorName}(context.Background(), clientOpt)
+        if err != nil {
+            t.Fatal(err)
+        }
+
+        var req {@test.requestTypeName}
+
+        # TODO(pongad): Use struct initializer instead of reflection.
```

Comment:
Ahh ok. Could you change the comment to be something like "Use the init code generator here"?